### PR TITLE
PIM-10343: Pull-up 6.0=> master

### PR DESCRIPTION
**Pull request by rfauglas**
**Description (for Contributor and Core Developer)**
Pull-up 6.0=> master

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php [hunk 1]
```diff
@@ -25,11 +25,11 @@ class DompdfBuilder implements PdfBuilderInterface
      */
     protected $dompdf;
 
-    protected HtmlFormatter $arabicHtmlFormatter;
+    protected ?HtmlFormatter $arabicHtmlFormatter;
 
     private string $publicDir;
 
-    public function __construct(string $rootDir, $publicDir, HtmlFormatter $arabicHtmlFormatter)
+    public function __construct(string $rootDir, $publicDir, ?HtmlFormatter $arabicHtmlFormatter = null)
```
Comment (nmarniesse) [inline src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php:32]:
```suggestion
    public function __construct(string $rootDir, $publicDir, HtmlFormatter $arabicHtmlFormatter)
```

File: src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php [hunk 2]
```diff
@@ -62,7 +62,10 @@ protected function render($html)
         ]);
         $this->dompdf = new Dompdf($options);
 
-        $html = $this->arabicHtmlFormatter->formatHtml($html);
+        if (null !== $this->arabicHtmlFormatter) {
```
Comment (nmarniesse) [inline src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php:65]: revert this if please

**Review by nmarniesse — APPROVED** [review]
GTM after fix

File: .circleci/workflows/editions/community.yml [hunk 3]
```diff
@@ -3,21 +3,6 @@ version: 2.1
...
-                  channel: ci
-                  webhook: $SLACK_NIGHTLY_STATUS
-                  fail_only: true
-
-    - &slack-post-step
-        post-steps:
-            - slack/status:
-                  channel: ci
-                  webhook: $SLACK_NIGHTLY_STATUS
-                  fail_only: false
```
Comment (StevenVAIDIE) [inline .circleci/workflows/editions/community.yml:19]: For me we should not remove those line in master @ahocquard and @nmarniesse what is you opinion ?

**Review by StevenVAIDIE — APPROVED** [review]

File: .circleci/workflows/editions/community.yml (hunk 3 above)
Comment (rfauglas) [reply .circleci/workflows/editions/community.yml]: I think you're right!
Reverted

Comment (nmarniesse) [reply .circleci/workflows/editions/community.yml]: Oh yes good catch!

**Pull request merged**